src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue
+    private Long id;
+    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
This type is default
```suggestion
    @OneToMany
```